pysat/tests/test_instrument.py

```diff
@@ -189,19 +189,6 @@ def test_instrument_init(self):
         """Test if init function supplied by instrument can modify object"""
         assert self.testInst.new_thing is True
 
-#     def test_getyrdoy_1(self):
-#         """Test the date to year, day of year code functionality"""
-#         date = pds.datetime(2009,1,1)
-#         yr, doy = pysat.utils.getyrdoy(date)
-#         assert ((yr == 2009) & (doy == 1))
-#
-#     def test_getyrdoy_leap_year(self):
-#         """Test the date to year, day of year code functionality
-#         (leap_year)"""
-#         date = pds.datetime(2008,12,31)
-#         yr, doy = pysat.utils.getyrdoy(date)
-#         assert ((yr == 2008) & (doy == 366))
-
     def test_custom_instrument_load(self):
```

Comment:
I think this method and the two below it may need to be in their own class. This whole suite of tests is called for xarray and other configs yet the modules are fixed to pysat_testing, so they aren't being run correctly for other situations. We can also move the reload to only handle these tests.

Reply:
Ideally the tests should be modified to accept the other testing instruments. Anytime I leave a gap in testing there is sure to be a bug.

Reply:
enhancement noted in issue #182